src/setup/scss/variables/_color.scss

```diff
@@ -8,6 +8,7 @@ $color-grayscale--400: #afafaf;
 $color-grayscale--500: #d8d8d8;
 $color-grayscale--600: #f3f3f3;
 $color-grayscale--700: #fcfcfc;
+$color-grayscale--800: #f5f5f5;
```

Comment:
suggestion: can we please use one of the already existing values, to keep this list as small as possible? I would even prefer if we could merge 600, 700, 800 into 1 single color value.

Reply:
@patric-eberle I can do that, but that will cause many changes in other components where we have used those variables that will be deleted if we merge them into a single variable. The merge value of 600, 700, 800 would be #f7f7f7.